### ISSUE TITLE
Problem with multi-state block group calls to get_acs()

### ISSUE THREAD

**Issue by NikKrieger** [walkerke/tidycensus#171]
In tidycensus 0.9.2, multistate calls to get_acs() at the tract and block group levels result in an error that really seems like a bug.

```
> library(tidycensus)
> get_acs("block group", variables = "B01003_001", year = 2016, state = c("de", "ct"))
Getting data from the 2012-2016 5-year ACS
  |====================================================================| 100%
Error: '10,09' is not a valid FIPS code or state name/abbreviation
In addition: Warning messages:
1: In if (grepl("^[[:digit:]]+$", state)) { :
  the condition has length > 1 and only the first element will be used
2: In if (grepl("^[[:alpha:]]+", state)) { :
 
 Error: '10,09' is not a valid FIPS code or state name/abbreviation
```

**Comment by walkerke** [comment]
I'm not getting the error for tracts but I do see it for block groups; I'll take a look as I agree it's a bug.

**Comment by walkerke** [comment]
This is fixed as of tidycensus 0.9.5.
